content/200-concepts/100-components/02-prisma-client/250-tracing.mdx

```diff
@@ -0,0 +1,362 @@
+---
+title: 'Tracing'
+metaTitle: 'Tracing (Preview)'
+metaDescription: 'Diagnose application performance with detailed traces of each query.'
+preview: true
+tocDepth: 3
+---
+
+<TopBlock>
+
+Prisma tracing logs
+Prisma tracing outputs
+
+Prisma tracing provides a detailed log of the activity that Prisma Client carries out, at an operation level, including the time taken to execute each query. It helps you analyze your application's performance and identify bottlenecks. Tracing is fully compliant with [OpenTelemetry](https://opentelemetry.io/), so you can use it as part of your end-to-end application tracing setup.
+
+<Admonition type="info">
+
+Tracing gives you a highly detailed, operation-level insight into your Prisma project. If you want aggregated numerical reporting, such as query counts, connection counts, and total query execution times, see [Metrics](/concepts/components/prisma-client/metrics).
+
+</Admonition>
+
+</TopBlock>
+
+## About tracing
+
+When you enable tracing, Prisma Client generates a trace for each operation that it makes. Each trace includes one or more [spans](https://www.opentelemetry.io/docs/reference/specification/trace/api/#span). Each span represents the length of time that one stage of the operation takes, such as a database statement or command, or serialization. Span durations are the round trip connection time from Prisma Client to the database and back.
```

Comment:
Last sentence here is weird. Span duration is always the duration the code takes that the span is wrapping/including I think. Maybe "Trace duration" instead of "Span duration" is meant here, so the total (or highest level span)?